bg.js

```diff
@@ -1,17 +1,23 @@
 let versions = null;
 let FIREFOX_VERSION_URL = "https://product-details.mozilla.org/1.0/firefox_versions.json";
 let BUGZILLA_PARAMS = [
-  "priority=--",
-  "f1=triage_owner",
-  "o1=equals",
-  // Note: v1 is inserted as the user email.
-  "resolution=---",
-  "include_fields=id", // this makes the query much faster.
   "chfield=%5BBug%20creation%5D",
-  "chfieldfrom=-60d", // only show bugs created up to 60 days ago;
+  "chfieldfrom=-60d", // sixty days previous
+  "chfieldto=Now",
+  "f1=triage_owner",
   "f3=keywords",
+  "f4=bug_severity",
+  "include_fields:=id", // only return ids since it's faster and we're just counting bugs
```

Comment:
Is the : at the end of include_fields valid?

Reply:
No, it's not. It's silently ignored, so I need to remove the ":"